shell/browser/ui/win/electron_desktop_window_tree_host_win.cc

```diff
@@ -133,4 +133,10 @@ void ElectronDesktopWindowTreeHostWin::OnNativeThemeUpdated(
   }
 }
 
+bool ElectronDesktopWindowTreeHostWin::ShouldWindowContentsBeTransparent()
+    const {
+  return native_window_view_->GetOpacity() < 1.0 ||
```

Comment:
Having a comment with a link to this PR will be very helpful for future maintenance.